src/components/editor_flexbox.tsx

```diff
@@ -1,83 +1,133 @@
 import SelectWrapper from './select_wrapper';
 import TextInput from './text_input';
+import MdxTemplateView from '../components/mdx_template_view'
+import { StoreContext, StoreProvider } from './store';
+import MdxWrapper from './mdx_wrapper';
+import Tab from 'react-bootstrap/Tab'
+import Tabs from 'react-bootstrap/Tabs'
+import {runSync, run} from '@mdx-js/mdx'
+import {evaluateSync,EvaluateOptions,compileSync} from '@mdx-js/mdx'
+// import {Fragment as _Fragment, jsx as _jsx} from 'react/jsx-runtime'
+import * as runtime from 'react/jsx-runtime' // Development.
+import {Fragment as _Fragment, jsx as _jsx, jsxs as _jsxs} from 'react/jsx-runtime'
+import * as provider from '@mdx-js/react'
+import { useMDXComponents } from '@mdx-js/react';
+import PhotoWrapper from './photo_wrapper';
+import TableWrapper from './table_wrapper';
+import TextInputWrapper from './text_input_wrapper';
+import DateStr from './date';
 
-//const md = `<StringInput label="Street Address" path="location.street_address" /><StringInput label="City" path="location.city" /><USStateSelect label="State" path="location.state" />`
-
-
-
-const EditorFlexBox = () => {
-
-  const [text, setText] = useState('');
+const components = {
+  Collapsible,
+  DateInput: DateInputWrapper,
+  Figure: FigureWrapper,
+  NumberInput: NumberInputWrapper,
+  Photo: PhotoWrapper,
+  PhotoInput: PhotoInputWrapper,
+  PrintSection,
+  Select: SelectWrapper,
+  StringInput: StringInputWrapper,
+  Table: TableWrapper,
+  TextInput: TextInputWrapper,
+  USStateSelect: USStateSelectWrapper,
+  DateStr: DateStr,
+  Tab: Tab,
+  Tabs: Tabs,
+};  
 
-  const handleChange = (event: ChangeEvent<HTMLTextAreaElement>) => {
-    setText(event.target.value);
-    event.target.style.height = 'auto'; // Reset the height to auto to adjust the size
-    event.target.style.height = `${event.target.scrollHeight}px`; // Set the height to the scrollHeight to adjust the size
-  };
 
+const EditorFlexBox: FC = () => {
+  const initialState = `
+  <br></br>
+  <PrintSection label="Print Report">
+  <Collapsible header="Collapsible — Instructions">
+      Here is an example of a collapsible component
+      1. First item
+      2. Second item
+      3. Third item
+  </Collapsible>
+  <Figure src="/images/hpwh/hpwh_components.jpeg">
+    The components in a common HPWH (BASC 2015)
+  </Figure>
+  <PhotoInput id="photo_input" label="Photo Input">
+    Provide a photo
+  </PhotoInput>
+  <NumberInput label="Number Input" min={0} path="number_input.value"  />
+  Value:{props.doc.number_input?.value}
+  <StringInput label="String Input" path="string_input.value" />
+  Value:{props.doc.string_input?.value}
+  </PrintSection>
+  <USStateSelect label="State" path="state_select.value" />
+  Value:{props.doc.state_select?.value}
+  `
+  const [textAreaContent, setTextAreaContent] = useState(initialState); // Declare a state variable...
+  
+  let {default: Content}  = evaluateSync(textAreaContent, {
```

Comment:
I suggest abstracting this into a function (don't forget to provide jsDoc documentation for it):
```
function generateTemplateView(templateText) {
...
return templateView
}
```
Then this function can be called to generate the initial view AND the updated view.